### PR TITLE
Feat: add dataverse catalog grid

### PR DESCRIPTION
**Pull request by lolottetheclash**
👉 This PR implements the Dataverse page grid.
It follows these specifications: https://github.com/okp4/dataverse-portal/issues/66

The `See Details` buttons are inactive for the moment, so they follow the design system style `disabled`.

The [react-markdown](https://www.npmjs.com/package/react-markdown) library has been added in order to offer the user the possibility to add description directly in markdown and be correctly rendered on screen.

🤔 Modifications had to be made to the `button` component in order to meet the expectations of the feature.
However, since buttons are used everywhere in the application, it would be important to take the time to look at the correct implementation of the `button` in order to add all the properties required for the button (size etc...) in order to avoid a too important refactor afterwards (with risk of regressions).

The `tag` component was also built to match the expectations of the feature. It will also be necessary to take the time to develop it in order to respect the options of the model.

### CODE REVIEW

File: src/page/dataverse/dataverse.tsx [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+  type: DataverseItem
+  label: string
+  description: string
+}
+
+const dataverseItems: DataverseItemDetails[] = [
+  {
+    type: 'dataspace',
+    label: 'AG Open Data Space',
+    description: 'Lorem ipsum dolor sit amet, consectetur '
```
Comment (ccamel) [inline src/page/dataverse/dataverse.tsx:23]: Note (not explicitly stated in the specification):
- Descriptions can contain any character, including line breaks.
- In addition, I would like to offer the ability to use [Markdown](https://www.markdownguide.org/getting-started/) in descriptions to provide text formatting options, possible image inclusions, as well as the ability to define HTML links.

Comment (ccamel) [reply src/page/dataverse/dataverse.tsx:23]: Great. Good choice for [remarkjs/react-markdown](https://github.com/remarkjs/react-markdown). :+1:

**Review by ccamel — APPROVED** [review]
That sounds great to me. :+1:  Thanks for adding markdown support to the descriptions. It's very promising. 😌

**Review by fredericvilcot — APPROVED** [review]
Great work ❤️

File: src/page/i18n/index.ts
[the remark this answers is not in the document]
Comment (lolottetheclash) [reply src/page/i18n/index.ts:1]: absolutely !!!

File: src/main.scss [hunk 2]
```diff
@@ -2,10 +2,9 @@
 @import '@/style/fonts.scss';
 @import '@/style/mixins.scss';
 
-@mixin blurred-page($left-position) {
-  position: absolute;
+@mixin blurred-page() {
```
[the remark this answers is not in the document]
Comment (lolottetheclash) [reply src/main.scss:5]: Done, with a `length` parameter👌 !

**Pull request merged**